### PR TITLE
Chinese formatDocument.label wrong translation fix

### PR DESCRIPTION
**Pull request by xiaohuilam**
translation improvement for `formatDocument.label`

### CODE REVIEW

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 1]
```diff
@@ -365,7 +365,7 @@
 			"hint1n": "第 {0} 行到第 {1} 行间进行了 1 次格式编辑",
 			"hintnn": "第 {1} 行到第 {2} 行间进行了 {0} 次格式编辑",
 			"no.provider": "当前没有安装“{0}”文件的格式化程序。",
-			"formatDocument.label": "设置文档的格式",
```
Comment (xiaohuilam) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: `设置文档的格式` means `set file type for document`!

Comment (lukaschal) [reply i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: On linguistic review.

Comment (lukaschal) [reply i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Sorry for mistake. Suggested TGT "格式化文档" was implemented as suggested 4 days ago.

**Comment by lukaschal** [comment]
@xiaohuilam Please verify and close.

**Comment by xiaohuilam** [comment]
> @xiaohuilam Please verify and close.

Latest translation is correct.

**Pull request closed without merging**